src/otx/core/model/entity/detection.py

```diff
@@ -143,3 +144,65 @@ def _customize_outputs(
             bboxes=bboxes,
             labels=labels,
         )
+
+
+class OVDetectionCompatibleModel(OTXDetectionModel):
+    """Object detection model compatible for OpenVINO IR inference.
+
+    It can consume OpenVINO IR model path or model name from Intel OMZ repository
+    and create the OTX detection model compatible for OTX testing pipeline.
+    """
+
+    def __init__(self, config: DictConfig) -> None:
+        self.model_name = config.pop("model_name")
+        self.config = config
+        super().__init__()
+
+    def _create_model(self) -> nn.Module:
+        from openvino.model_api.models import DetectionModel
+
+        model = DetectionModel.create_model(self.model_name)
+        if model.get_label_name(0) == "background":
```

Comment:
```suggestion
        if model.get_label_name(0).lower() == "background":
```
I suggest to use `lower()` function to cover the "Background" case.